naresh/src/main/java/com/lftechnology/phpjava/assignmentthree/EvenFibonacci.java

```diff
@@ -0,0 +1,50 @@
+package com.lftechnology.phpjava.assignmentthree;
+
+import com.lftechnology.phpjava.menu.MenuReader;
+import com.lftechnology.phpjava.menu.ProgramMenu;
+
+import java.util.Scanner;
+
+/**
+ * Created by naresh on 5/16/16.
+ */
+public class EvenFibonacci implements MenuReader {
+    /**
+     * read the user input for the action
+     */
+    @Override
+    public void scanner() {
+
+        Scanner scan = new Scanner(System.in);
+        System.out.println("Please enter the max value.");
+        String maxValue = scan.nextLine();
+        int max = Integer.valueOf(maxValue);
+        int counter = 0;
+        int sum = 0;
+        int a, b, c;
+        a = 1;
+        b = 2;
+
+        while (counter < max) {
+            c = a + b;
+            a = b;
+            b = c;
+            counter = b;
+            if (counter > max) {
+                break;
+            }
+            if (counter % 2 == 0) {
+                sum = sum + counter;
+            }
+        }
+        System.out.println("Sum of fibonacci numbers is :" + sum);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS106)